src/util/expr.cpp

```diff
@@ -327,6 +327,24 @@ void exprt::visit(const_expr_visitort &visitor) const
   }
 }
 
+void exprt::visit(const std::function<void(const exprt &)> &f) const
```

Comment:
Ok, but we should really have this in a separate commit, it would in fact warrant both documentation and a unit test.